src/ontoweaver/transformer.py

```diff
@@ -46,22 +47,38 @@ def __call__(self, row, i):
 
 
 class split(base.Transformer):
-    """Transformer subclass used to split cell values at defined separator and create nodes with
+    """Transformer subclass used to split cell values at defined separator and label_maker nodes with
     their respective values as id."""
 
-    def __init__(self,properties_of, branching_properties = None, columns=None, output_validator: validate.OutputValidator = None, raise_errors = True, **kwargs):
+    class ValueMaker(make_value.ValueMaker):
+
+        def __init__(self, raise_errors: bool = True, separator: str = None):
+            self.separator = None
+            super().__init__(raise_errors)
+
+        def select(self, columns, row, i):
+
+            for key in columns:
+                items = str(row[key]).split(self.separator)
+                for item in items:
+                    yield item
+
+    def __init__(self, properties_of, value_maker = ValueMaker(), label_maker = None, branching_properties = None, columns=None, output_validator: validate.OutputValidator = None, raise_errors = True, **kwargs):
```

Comment:
We should have an explicit declaration of the default arguments in the constructor's signature…
```suggestion
    def __init__(self, properties_of, value_maker = ValueMaker(), label_maker = None, branching_properties = None, columns=None, output_validator: validate.OutputValidator = None, raise_errors = True, separator = ",", **kwargs):
```